extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -190,14 +200,8 @@ export class AmpGeo extends AMP.BaseElement {
         }
         this.mode_ = mode.GEO_OVERRIDE;
       }
-    } else if (
-      preRenderMatch &&
-      !Services.urlForDoc(this.element).isProxyOrigin(this.win.location)
-    ) {
-      // pre-rendered by a publisher case, if we're a cache we ignore that
-      // since there is no way the publisher could know the geo of the client.
-      // When caches start pre-rendering geo we'll need to add specifc code
-      // to handle that.
+    } else if (preRenderMatch && isExperimentOn('amp-geo-ssr')) {
```

Comment:
@jridgewell could we get away with not guarding this with `isExperimentOn`? this is why i wanted to add a special `i-amp-geo-ssr` class to detect. 

i could also do a `prerenderHtmlMatch` if we can assume that having geo classes on html is a sign that we ssr'd it

Reply:
Ahh, I see. We don't really need the experiment, we just need to know that the Pub's geo classes were stripped. I think this is fine for now.

```suggestion
    } else if (preRenderMatch && (!Services.urlForDoc(this.element).isProxyOrigin(this.win.location) || isExperimentOn('amp-geo-ssr'))) {
```